package ERASE_MONSTER;

message CONTENTS
{
	message char_info
	{
		optional int32 id = 1;
	}
	repeated char_info data = 1; 
}